doc/source/_static/css/getting_started.css

```diff
@@ -236,11 +236,12 @@ ul.task-bullet > li > p:first-child {
 
 .tutorial-card .card-header {
   cursor: pointer;
-  background-color: white;
+  background-color: transparent;
 }
 
 .tutorial-card .card-body {
   background-color: #F0F0F0;
```

Comment:
Can you remove this line please?

Reply:
sure

Reply:
the change isn't being acknowledged as a modification, so it's not commiting (working tree clean)